env.d.ts

```diff
@@ -725,8 +725,11 @@ type MongoTypeNumber = -1 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 1
 
 type MongoValue = string | number | boolean | Date | MongoValue[] | { [key: string]: MongoValue } | null
 
-type MongoCommandValue = string | number | boolean | Date | MongoCommandValue[] | { [key: string]: MongoCommandValue } |
-null | undefined
+type MongoCommandValue = string | number | boolean | Date | MongoCommandValue[] | { [key: string]: MongoCommandValue } | null | undefined;
```

Comment:
Please can you undo your changes to this line so it doesn't show up in the "Files changed" diff.